File: src/main/java/fi/finlit/edith/sql/domain/NoteType.java
package fi.finlit.edith.sql.domain;

import javax.persistence.Table;

@Table(name = "note_type")
public enum NoteType {
    WORD_EXPLANATION,
    LITERARY,
    HISTORICAL,
    DICTUM,
    CRITIQUE,
    TITLE,
    TRANSLATION,
    REFERENCE
}

File: src/main/scala/fi/finlit/edith/sql/domain/NoteType.scala
package fi.finlit.edith.sql.domain

//remove if not needed
import scala.collection.JavaConversions._

object NoteType {

  class NoteType {
  }

  val WORD_EXPLANATION = new NoteType()

  val LITERARY = new NoteType()

  val HISTORICAL = new NoteType()

  val DICTUM = new NoteType()

  val CRITIQUE = new NoteType()

  val TITLE = new NoteType()

  val TRANSLATION = new NoteType()

  val REFERENCE = new NoteType()
}
